### PR TITLE
Controller web use json string create and update schemas 

### PR DESCRIPTION
**Pull request by ly923976094**
Controller web cannot create and update schemas in /schemas and /schemas/{schemaName}，Because these two interfaces need to upload a schema file to create and update the schema, but the web can only pass json strings, so they are deprecated and added services that can pass json strings to create and update the schema(/schemas/v2 and /schemas/v2/{schemaName}).

### CODE REVIEW

File: pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java [hunk 1]
```diff
@@ -109,6 +109,7 @@ public SuccessResponse deleteSchema(
     return new SuccessResponse("Schema " + schemaName + " deleted");
   }
 
+  @Deprecated
```
Comment (sunithabeeram) [inline pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java]: I am not sure if we can deprecate this, as I believe swagger UI uses this path (see https://pinot.readthedocs.io/en/latest/management_api.html#pinot-management-console).

**Comment by sunithabeeram** [comment]
A couple of general notes: please always add unit and integration tests to ensure:
a) Your code works as expected.
b) Your code is future proof-ed against refactors/regressions.

This section in the contribution guide expects manual testing done to be listed in the PR description:
https://pinot.readthedocs.io/en/latest/contribution_guidelines.html#creating-a-pull-request-pr

(I will update the contribution guide to include in the description the above 2 points I brought up as well)

File: pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java (hunk 1 above)
Comment (ly923976094) [reply pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java]: The fact that it is deprecated does not mean that its functionality cannot be used. This method can still be used in places such as script creation schema, but it is suggested that this method cannot be used in controller web. Some tests will be added later

**Comment by ly923976094** [comment]
> A couple of general notes: please always add unit and integration tests to ensure:
> a) Your code works as expected.
> b) Your code is future proof-ed against refactors/regressions.
> 
> This section in the contribution guide expects manual testing done to be listed in the PR description:
> https://pinot.readthedocs.io/en/latest/contribution_guidelines.html#creating-a-pull-request-pr
> 
> (I will update the contribution guide to include in the description the above 2 points I brought up as well)



> A couple of general notes: please always add unit and integration tests to ensure:
> a) Your code works as expected.
> b) Your code is future proof-ed against refactors/regressions.
> 
> This section in the contribution guide expects manual testing done to be listed in the PR description:
> https://pinot.readthedocs.io/en/latest/contribution_guidelines.html#creating-a-pull-request-pr
> 
> (I will update the contribution guide to include in the description the above 2 points I brought up as well)

Ok, we'll add the corresponding unit tests later

File: pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java (hunk 1 above)
Comment (ly923976094) [reply pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java]: The fact that it is deprecated does not mean that its functionality cannot be used. This method can still be used in places such as script creation schema. it is suggested that this method cannot be used in controller web.

**Pull request closed without merging**